src/EmbeddingOpts.scala

```diff
@@ -1,56 +0,0 @@
-//package cc.factorie.app.nlp.embeddings
-import cc.factorie.util.CmdOptions
-
-class EmbeddingOpts extends CmdOptions {
```

Comment:
Opts passed to each model class to configure things such as optimizer learning rate, negative sampling rate, etc.